src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -176,8 +224,8 @@ protected override void Append(StringBuilder builder, LogEventInfo logEvent)
         /// Appends the Message of an Exception to the specified <see cref="StringBuilder" />.
         /// </summary>
         /// <param name="sb">The <see cref="StringBuilder"/> to append the rendered data to.</param>
-        /// <param name="ex">The exception containing the Message to append.</param>
-        protected virtual void AppendMessage(StringBuilder sb, Exception ex)
+        /// <param name="ex">The exception containing the Message to append.</param>        
+        private static void AppendMessage(StringBuilder sb, Exception ex)
```

Comment:
> Atleast for now and them remove the virtual methods in upcoming releases.

Doesn't sound future proof to me. Well, mabye was the dictionary (my) mistake and not a good step?